Makefile

```diff
@@ -91,8 +92,11 @@ install: setup.data rbac_static.csv
 	scripts/install.sh 755 block_device_io.native $(DESTDIR)$(LIBEXECDIR)/block_device_io
 # Libraries
 	ocaml setup.ml -install
-# xenserver html docs
+# xenserver docs
 	mkdir -p $(DESTDIR)$(DOCDIR)/html/xenserver
 	cp -r -L _build/ocaml/doc/api $(DESTDIR)$(DOCDIR)/html/xenserver
 	cd ocaml/doc && cp *.js *.html *.css *.png $(DESTDIR)$(DOCDIR)/html/xenserver
 	cp _build/ocaml/doc/branding.js $(DESTDIR)$(DOCDIR)/html/xenserver/branding.js
+	mkdir -p $(DESTDIR)$(DOCDIR)/pdf/xenserver
```

Comment:
I think these directories could also be renamed to `markdown`, unless we want to specifically generate a PDF out of it.